packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -561,6 +613,7 @@ export class PerformanceEvent {
 /**
  * Logger that is useful for UT
  * It can be used in places where logger instance is required, but events should be not send over.
+ * @deprecated - Use createChildLogger instead
  */
 export class TelemetryUTLogger implements ITelemetryLoggerExt {
```

Comment:
Replacing this with `createChildLogger()` is a loss of behavior, right? The tests using it would stop reporting errors to console.

Reply:
we hide console logs by default in tests, so i'm not sure how much value that provides, i really just want all the test loggers out of this production package. we could copy this to where it is used, rather than using child loggers, but not sure it is worth it since we try to avoid console logs in tests

Reply:
(I never liked this class - I think it's better to use `MockLogger` and be explicit about failing the test, rather than blindly throwing and hoping it breaks whatever you're asserting in the test)

Reply:
Ok but responding to the actual comment - `reportError` also throws.  So it's a pretty big change, I wouldn't do it here.

Reply:
i missed that that logger threw, and didn't just  output it, i will revert these changes, as not throwing is a regression